03-rnaseq/differential-expression_rnaseq_01.Rmd

```diff
@@ -249,61 +255,75 @@ head(metadata$refinebio_title)
 
 ## Set up metadata
 
-This dataset includes data from patients with and without ASXL gene mutations. 
-The authors of this data have ASXL mutation status along with other information is stored all in one string (this is not very convenient for us). 
+This dataset includes data from mouse lymphoid cells with human RPL10, with and without a `R98S` mutation.
+The authors of this data have the mutation status along with other information stored all in one string (this is not very convenient for us). 
```

Comment:
The only reason I do this (admittedly kind of blamey thing)  is to try to shed light on the idea that how you submit your metadata matters because we at refine.bio can only do certain things with it. (And it's not refine.bio's fault).

Reply:
I use your wording anyway.